src/site/antora/modules/ROOT/examples/manual/flowtracing/log4j2.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to you under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<Configuration xmlns="https://logging.apache.org/xml/ns"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xsi:schemaLocation="
+                   https://logging.apache.org/xml/ns
+                   https://logging.apache.org/xml/ns/log4j-config-2.xsd">
+
+  <MarkerFilter marker="FLOW" onMatch="ACCEPT" onMismatch="NEUTRAL"/><!--1-->
+
+  <Appenders>
+    <Console name="CONSOLE">
+      <PatternLayout>
+        <MarkerPatternSelector defaultPattern="%d %5p [%t] %c{1} - %m%n"><!--2-->
+          <PatternMatch key="ENTER" pattern="%d %5p [%t] %c{1} → %m%n"/><!--3-->
```

Comment:
Graphically `→` is barely noticeable. Maybe we should:

- either choose a bigger Unicode arrow,
- or use `=>` instead.

The latter is used for example by Exim.

Reply:
Fixed in 3684d63934526d51633a0f69e3cccc6eedd94011.